styles/styles.css

```diff
@@ -0,0 +1,565 @@
+* {
+  margin: 0 0;
+  padding: 0 0;
+}
+
+body {
+  background-color: #f4f5f6;
+}
+
+ul {
+  list-style: none;
+}
+
+p,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  line-height: 1.1;
+  font-family: 'Roboto Condensed', sans-serif;
+}
+
+a {
+  color: inherit;
+  text-decoration: none;
+}
+
+a:hover {
+  color: inherit;
+  text-decoration: none;
+  border-bottom: 1px solid #f72210;
+}
+
+ul li {
+  display: inline-block;
+  margin: 0;
+  padding: 0;
+}
```

Comment:
You can use this bootstrap styling `<li class="d-none m-0 p-0">` for line 36-40.

Reply:
The above are a few examples. Most of your CSS styling could easily be done by bootstrap. Please review it.